kernelci.org/content/en/blog/posts/2021/hackfests.md

```diff
@@ -0,0 +1,135 @@
+test, say in kselftest.  A second participant enables the test to run in
+KernelCI, which fails in some cases and a kernel bug is found.  A third
+participant makes a fix for the bug, which can then be tested directly in
+KernelCI to confirm it works as expected.  This may even happen on
+[staging.kernelci.org](https://staging.kernelci.org) before the patches for the
+test and the fix are sent to any mailing list, in which case the fix would get
+a `Tested-by: "kernelci.org bot" <bot@kernelci.org>` trailer from the start.
+This scenario also relies on some hardware previously made available in test
+labs by other people, putting together efforts from at least four different
+participants.
+
+
+## Timeline
+
+Here's a summary of the first two hackfests:
+
+### Hackfest #1 - 27th May to 4th June 2021
+
+* Workboard: https://github.com/orgs/kernelci/projects/3
+* Participans
+  * Several kernel developers from [Google Chrome
+  OS](https://www.google.com/intl/en_uk/chromebook/chrome-os/)
+  * Several kernel developers from [Collabora](https://collabora.com)
+  * A few members of the core KernelCI team
+* Achievements
+  * New KernelCI instance for Chrome OS on https://chromeos.kernelci.org
+  * Added support for building Chrome OS configs on top of mainline
+  * Enabled clang-13 builds for Chrome OS and main KernelCI builds
+  * New test suite for libcamera enabled on https://linux.kernelci.org
+  * Enabled LTP crypto tests with extra kernel config fragment
+  * Several patches were also sent to extend kselftest and KUnit coverage in
+      the kernel tree
+
+### Hackfest #2 - 6th to 10th September 2021
+
+* Workboard: https://github.com/orgs/kernelci/projects/5
+* Participants
+  * More members of the [TSC](/docs/org/tsc/) took part than first hackfest
+  * New contributor: Denis Efremov (floppy disk kernel maintainer)
+  * Alice from [Cybertrust Japan](https://www.cybertrust.co.jp/english/)
+  * Chris from [Civil Infrastructure Platform](https://www.cip-project.org/)
+    (CIP)
+  * Nikolai from [Red Hat's CKI](https://cki-project.org/)
+  * Several KernelCI developers from [Collabora](https://collabora.com)
+* Achievements
+  * New Docker images and initial steps for running
+      [`cvehound`](https://github.com/evdenis/cvehound)
+  * On-going progress with Chrome OS tests using QEMU and hardware in LAVA
+  * Support for CIP kernel configs
+  * Initial work to enable CIP Core images (Debian based)
+  * Set up redirect from [cip.kernelci.org](https://cip.kernelci.org) to the
+    web dashboard page with [CIP jobs](https://linux.kernelci.org/job/cip/)
+  * Support for `xilinx-zcu102` device
+  * Initial work to enable IEC cybersecurity tests
+  * Initial work to enable kselftest livepatch
+  * Most of the content from https://foundation.kernelci.org replicated on the
+    new [kernelci.org](https://kernelci.org) static site
+  * Started adding support for including kernel firmware files in test jobs
```

Comment:
Ah I didn't realise this - fantastic.

There are some Renesas configs that need USB firmware to avoid a massive 60s timeout on boot...

Reply:
Yes, the plan is to have a list of firmware files in to include in the rootfs jobs initially in `rootfs-configs.yaml`.  So we'll pick a few to get the best size/usefulness ratio with the devices currently enabled in KernelCI.